paywall/src/data-iframe/index.2.0.ts

```diff
@@ -2,14 +2,49 @@ import Postmate from 'postmate'
 import { Web3Service } from '@unlock-protocol/unlock-js'
 import { BlockchainReader } from './BlockchainReader'
 import { BlockchainDataStorable } from './BlockchainDataStorable'
-import configure from '../config'
+
+declare var __ENVIRONMENT_VARIABLES__: any
+
+// TODO FIND A WAY TO MOVE THIS IN A BETTER PLACE
+// NOTE: NEVER PUT ENV VARS HERE
+const constants: { [key: string]: any } = {
```

Comment:
This is an artifact from the previous implementation. I think we can sort this out. It _might_ be worth splitting out the various iframes into different subrepos (or at least into one `paywall-iframes` subrepo), which would allow us to keep the configuration specific to them in one place.